src/Illuminate/Support/helpers.php

```diff
@@ -128,6 +128,10 @@ function e($value, $doubleEncode = true)
             $value = $value->value;
         }
 
+        if (is_int($value) || is_float($value)) {
+            return (string) $value;
+        }
```

Comment:
breaking change

Reply:
Why? Because `return` without passing to `htmlspecialchars` function?

Reply:
Maybe can be replaced with 

```php
return htmlspecialchars((string) $value, ENT_QUOTES | ENT_SUBSTITUTE, 'UTF-8', $doubleEncode);
    }
```

Reply:
Malformed numbers would throw an exception -- the bigger question is is this even needed?

Laravel returns a string regardless.

Reply:
I think that convert an implicit cast to an explicit it's better for a typification perspective.

Behind the scenes, however, a cast from number to string takes place... so a compromise could be:
```php
return htmlspecialchars((string) $value, ENT_QUOTES | ENT_SUBSTITUTE, 'UTF-8', $doubleEncode);
```